### PR TITLE
Add EOE player cards from FFG stream

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/eoe/eoep.json [hunk 1]
```diff
@@ -1,187 +1,595 @@
...
+        "type_code": "asset"
+    },
+    {
+        "back_link": "08013b",
+        "code": "08013a",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Lin Hsiang",
+        "is_unique": true,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:188]: Not unique.

File: pack/eoe/eoep.json [hunk 2]
```diff
@@ -1,187 +1,595 @@
...
+        "traits": "Unbroken.",
+        "type_code": "asset"
+    },
+    {
+        "code": "08013b",
+        "cost": null,
+        "faction_code": "neutral",
+        "hidden": true,
+        "illustrator": "Lin Hsiang",
+        "is_unique": true,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:206]: Not unique.

File: pack/eoe/eoep.json [hunk 3]
```diff
@@ -1,187 +1,595 @@
...
+        "name": "Discipline",
+        "pack_code": "eoep",
+        "permanent": true,
+        "position": 13,
+        "quantity": 1,
+        "subname": "Prescience of Faith",
+        "subtype_code": "weakness",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:213]: Again, not a weakness... but yeah.

File: pack/eoe/eoep.json [hunk 4]
```diff
@@ -1,187 +1,595 @@
...
+        "type_code": "asset"
+    },
+    {
+        "back_link": "08014b",
+        "code": "08014a",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Lin Hsiang",
+        "is_unique": true,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:225]: Not unique.

File: pack/eoe/eoep.json [hunk 5]
```diff
@@ -1,187 +1,595 @@
...
+        "traits": "Unbroken.",
+        "type_code": "asset"
+    },
+    {
+        "code": "08014b",
+        "cost": null,
+        "faction_code": "neutral",
+        "hidden": true,
+        "illustrator": "Lin Hsiang",
+        "is_unique": true,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:243]: Not unique.

File: pack/eoe/eoep.json [hunk 6]
```diff
@@ -1,187 +1,595 @@
...
+        "name": "Discipline",
+        "pack_code": "eoep",
+        "permanent": true,
+        "position": 14,
+        "quantity": 1,
+        "subname": "Balance of Body",
+        "subtype_code": "weakness",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:250]: NB weakness.

File: pack/eoe/eoep.json [hunk 7]
```diff
@@ -1,187 +1,595 @@
...
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "flavor": "Worshipped by its own cult, one might say.",
+        "illustrator": "Tiziano Baracchi",
+        "name": "Greed",
+        "pack_code": "eoep",
+        "position": 18,
+        "quantity": 1,
+        "restrictions": "investigator:08016",
+        "skill_intellect": 2,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:335]: No skill line.

File: pack/eoe/eoep.json [hunk 8]
```diff
@@ -1,187 +1,595 @@
...
+        "skill_wild": 1,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:336]: No skill line.

File: pack/eoe/eoep.json [hunk 9]
```diff
@@ -1,187 +1,595 @@
...
+        "code": "08048",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "illustrator": "Vlad Ricean",
+        "name": "21 or Bust",
+        "pack_code": "eoep",
+        "position": 48,
+        "quantity": 2,
+        "skill_wild": 1,
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:405]: skill_combat: 1 / skill_agility: 1 (delete skill_wild: 1)

File: pack/eoe/eoep.json [hunk 10]
```diff
@@ -1,187 +1,595 @@
...
+        "text": "One at a time, reveal random tokens from the chaos bag until you choose to stop. Treat each [skull], [cultist], [tablet], or [elder_thing] as a 5; the [auto_fail] as a 10; and the [elder_sign] as either a 1 or an 11. If the combined value of those tokens (ignoreing +/-) is...\n- ...18 or less, gain 4 resources.\n- ...19, gain 5 resources.\n- ...20, gain 6 resources.\n- ...21, gain 9 resources.",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:406]: Typo: 'ignoring' for 'ignoreing'.

File: pack/eoe/eoep.json [hunk 11]
```diff
@@ -1,187 +1,595 @@
...
+        "xp": 1
+    },
+    {
+        "code": "08053",
+        "cost": 2,
+        "deck_limit": 1,
+        "exceptional": true,
+        "faction_code": "rogue",
+        "illustrator": "Robert Laskey",
+        "name": "The Red Clock",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:435]: is_unique: true

File: pack/eoe/eoep.json [hunk 12]
```diff
@@ -1,187 +1,595 @@
...
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Patrick McEvoy",
+        "name": "Schoffner's Catalogue",
+        "pack_code": "eoep",
+        "position": 72,
+        "quantity": 2,
+        "skill_intellect": 1,
+        "text": "Uses (5 secrets). If Schoffner's Catalogue has no secrets, discard it.\nYou may spend secrets on Schoffner's Catalogue as resources to pay for [[Item]] assets played by an investigator at your location.",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:493]: 'any investigator', not 'an investigator' at the end of the line.

**Review by drawntotheflame — COMMENTED** [review]
Great job, @zzorba , impressive stuff!

**Comment by zzorba** [comment]
Okay, addressed all the feedback.  @Kamalisk I think this can go live.

**Pull request merged**